tracer/src/Datadog.Trace/Logging/DirectSubmission/Formatting/LogFormatter.cs

```diff
@@ -63,7 +62,7 @@ public LogFormatter(
 
         internal delegate LogPropertyRenderingDetails FormatDelegate<T>(JsonTextWriter writer, in T state);
 
-        internal string? Tags { get; private set; }
+        private string? Tags { get; set; }
```

Comment:
Looks like this needs to be `internal`


```suggestion
        internal string? Tags { get; set; }
```